src/config/chains/telos-evm/index.ts

```diff
@@ -352,6 +353,10 @@ export default class TelosEVM extends EVMChainSettings {
         return config.headerIndicators;
     }
 
+    getIndexerSecondsBehindThreshold(): number {
+        return config.secondsBehindThreshold || 3;
```

Comment:
Let's remove the default and just throw an error if it isn't properly configured, what do you think?